src/hooks/useForm.ts

```diff
@@ -0,0 +1,41 @@
+import { ChangeEvent, useState } from "react"
+
+
+
+interface FormState<T> {
+    formState: T
+}
+
+
+function useForm<T>( initialState: T )  {
```

Comment:
You probably don't need the generics, this hook is only used in one file, you could move the types here and remove the generics